src/main/scala/org/ergoplatform/api/routes/TransactionsApiRoute.scala

```diff
@@ -26,10 +26,10 @@ case class TransactionsApiRoute(readersHolder: ActorRef, nodeViewActorRef: Actor
 
   override val settings: RESTApiSettings = restApiSettings
 
-  private def getMemPool: Future[Option[ErgoMemPoolReader]] = (readersHolder ? GetReaders).mapTo[Readers].map(_.m)
+  private def getMemPool: Future[ErgoMemPoolReader] = (readersHolder ? GetReaders).mapTo[Readers].map(_.m)
```

Comment:
looks like we should wait the whole timeout if MemPool was not ready yet at the moment of request.